posts/discriminative-vs-generative/index.qmd

```diff
@@ -0,0 +1,595 @@
+---
+title: "Generative vs. discriminative models: what's the difference really?"
+subtitle: "Thinking through the differences between generative and discriminative models."
+date: "2022/04/03"
+image: "params.png"
+categories: [factor graphs, ML, theory]
```

Comment:
um. not sure these are very helpful, but think this is a the-more-the-merrier type situation

Reply:
Had thought similar but can't think of many others atm. Maybe PGM? Don't think we'll want to add semi-supervised as the bulk of that stuff will go into the next blog